### PR TITLE
Feature/wkwebview fixes

### PR DESCRIPTION
**Pull request by santicarmo31**
image

### CODE REVIEW

File: Source/FolioReaderCenter.swift [hunk 1]
```diff
@@ -812,7 +812,9 @@ open class FolioReaderCenter: UIViewController, UICollectionViewDelegate, UIColl
             html = modifiedHtmlContent
         }
         
-        cell.loadHTMLString(html, baseURL: URL(fileURLWithPath: spine.resource.fullHref.deletingLastPathComponent))
+        let fileURL = URL(fileURLWithPath: spine.resource.fullHref)
```
Comment (ricardohg) [inline Source/FolioReaderCenter.swift:815]: maybe add some comment here explaining the double: load fileUrl + loadHtml, also why removing 2 last path components is needed

Comment (santicarmo31) [reply Source/FolioReaderCenter.swift:815]: Done

**Comment by ss0217** [comment]
Hi @ricardohg , @santicarmo31 
I did the migration to WKWebview and got stuck when there are multiple pages or large content inside the webview. Let's say if a chapter has 5-10 pages then the collectionViewCell only displays the first page inside the webview, rest of the content is getting cut.  Did you guys face such an issue? UIWebview had paginationMode and paginationBreakingMode properties which is currently unavailable in WKWebView.

**Comment by thilakks** [comment]
Hi @ricardohg , @santicarmo31 
   I'm facing issue with UWebview in Appstore  please update us WKWebview

**Comment by ThakarRajesh** [comment]
Hi @ricardohg , @santicarmo31
I'm facing issues with UWebview in Appstore please update us on WKWebview.app reject by apple.please give us a solution asap.

File: Source/FolioReaderCenter.swift (hunk 1 above)
Comment (chulkov) [reply Source/FolioReaderCenter.swift:815]: Thank you so much❤️

**Pull request merged**